### ISSUE TITLE
Pre-built Docker Image Not Publicly Available

### ISSUE THREAD

**Issue by hacdias** [uber-go/sally#124]
Readme states:

https://github.com/uber-go/sally/blob/686fb8782cfa89a807ecc6dcbc104c6dab43eb66/README.md?plain=1#L14-L15

That URL gives 404 to me. Tried with `docker pull`:

```console
docker pull ghcr.io/uber-go/sally:latest
Error response from daemon: Head "https://ghcr.io/v2/uber-go/sally/manifests/latest": unauthorized
```

Which leads me to believe that it is published, just not publicly available. Is there a reason why it's not publicly available?

cc @abhinav as I see you pushed this update (434fc43c4b5403b38db408621a286096806c4036).

**Comment by abhinav** [comment]
@hacdias That was definitely not intentional.
I don't have the appropriate privileges to dig into the visibility issues here.
@mway @alxn @sywhang @r-hang, would one of you be able to look into this?
Thanks!

**Comment by hacdias** [comment]
Is there any update on this? It still seems to be private.

**Comment by mway** [comment]
Hey @hacdias, sorry for the wait - we've updated the visibility for this, can you try again?

**Comment by hacdias** [comment]
@mway it's now visible. Thanks! Do you think it'd be possible to get a new release out with the latest changes?

**Comment by mway** [comment]
@hacdias I just released [v1.6.0](https://github.com/uber-go/sally/releases/tag/v1.6.0) ([container](https://github.com/uber-go/sally/pkgs/container/sally/159337128?tag=v1.6.0)) - let us know if you have issues. Thanks for your contributions!
